modules/c++/io/tests/serializeTest1.cpp

```diff
@@ -73,7 +73,9 @@ class A : public Serializable
     {
 	std::string toFill;
 	sys::byte b[1];
-	while (true)
+	// Work-around warning C4127: conditional expression is constant;
+	// see https://msdn.microsoft.com/en-us/library/6t66728h.aspx and http://stackoverflow.com/questions/1946445/c-c-how-to-use-the-do-while0-construct-without-compiler-warnings-like-c412
+	for (;;) // while (true)
```

Comment:
Rather than this, is there a `/wd4127` we can put in `build.py`?  `while (true)` is more instantly obvious to people what's going on I think.